src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosDriverFactory.scala

```diff
@@ -3,8 +3,13 @@ package org.apache.mesos.chronos.scheduler.mesos
 import java.util.logging.Logger
 
 import org.apache.mesos.chronos.scheduler.config.SchedulerConfiguration
-import org.apache.mesos.Protos.{FrameworkInfo, Status}
+import org.apache.mesos.Protos.{Credential, FrameworkInfo, Status}
 import org.apache.mesos.{MesosSchedulerDriver, Scheduler}
+import com.google.protobuf.ByteString
+import java.io.{ File, FileInputStream, IOException }
+import java.nio.file.{ Files, Paths }
```

Comment:
Style consistency: Looks like the other imports don't put whitespace inside the {}.
`s/{ Files, Paths }/{Files, Paths}/`